packages/kcov.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Kcov < Package
+  description 'Kcov is a code coverage tester for compiled programs, Python scripts and shell scripts.'
+  homepage 'http://simonkagstrom.github.io/kcov/'
+  version 'v36'
+  source_url 'https://github.com/SimonKagstrom/kcov/archive/v36.tar.gz'
+  source_sha256 '29ccdde3bd44f14e0d7c88d709e1e5ff9b448e735538ae45ee08b73c19a2ea0b'
+
+  depends_on 'curl'
+  depends_on 'elfutils'
+
+  def self.build
+    Dir.mkdir 'build'
+    Dir.chdir 'build' do
+      system 'cmake',
+             '-DCMAKE_BUILD_TYPE=Release',
+             "-DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX}",
```

Comment:
You need this line to get rid of the error in the log:
```
"-DLIBBFD_IBERTY_LIBRARY=#{CREW_LIB_PREFIX}/libbsd.so",
```

Reply:
The error in the log I'm referring to is this:
```
-- Could NOT find LIBBFD (missing: LIBBFD_IBERTY_LIBRARY) 
```

Reply:
There's a difference between `libbfd` and `libbsd`. The first is the Binary File Descriptor library. The other is a collection of miscellaneous functions not found in GNU libc.

Reply:
If we were going to pass that option, we'd need to pass `--enable-shared --enable-install-libbfd --enable-install-libiberty` in Binutils, and it wouldn't use `libbsd.so`, either. In this case, the error is correct.

Reply:
Yes, libbfd.so was the file.  That was a typo.  Your explanation makes sense.